pom.xml

```diff
@@ -151,7 +151,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
-        <version>3.6.4</version>
+        <version>3.10.2</version>
```

Comment:
1. should be taken from prent
2. we should check site generation - report plugin was changed to `maven-plugin-report-plugin`

Reply:
You are right, I missed this. 

I reverted commits.